src/main/resources/google_checks.xml

```diff
@@ -328,6 +328,13 @@
       <property name="tokens" value="METHOD_DEF, CTOR_DEF, ANNOTATION_FIELD_DEF,
                                    COMPACT_CTOR_DEF"/>
     </module>
+    <module name="MissingJavadocType">
+      <property name="scope" value="protected"/>
```

Comment:
Why protected ?
Can you quote a guide?

Reply:
https://github.com/checkstyle/checkstyle/pull/8987#discussion_r524343863